### PR TITLE
Replace invalid-UTF-8 smart quotes

### PR DESCRIPTION
**Pull request by lvh**
These bytes (0x92, 0x93), which appear to be intended as smart quotes, are invalid UTF-8, preventing it from being parsed by many tools. I have replaced them with single quotes (Unicode APOSTROPHE) to match the surrounding style. This is the only file with this defect.

Also cleaned up some end-of line whitespace.

I have found that most files do not end in newlines. Is that intentional?

Should you be interested in automated tooling that can help find this, here's the (fish, but will work in zsh and possibly recent bash) command I used to check that there was only one file:

```
iconv -f utf-8 -t utf-8 **json > /dev/null
```

I don't have a CLA; could you please send me in the right direction for submitting one?

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@lvh__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by lvh** [comment]
The build is failing on generating the compute client, which doesn't make a lot of sense to me. I could remove those commits, but it appears the build only looks at files in the PR, so that might simply silence the failure... Could someone else take a look to confirm? It looks like it might just be downloading the wrong thing:

https://travis-ci.org/Azure/azure-rest-api-specs/jobs/140436786#L250

**Comment by lmazuel** [comment]
Hi @lvh 

The build breaks because of this bug:
https://github.com/Azure/azure-rest-api-specs/issues/352

The current compute file is invalid, but no-one has produced a PR with the fix right now, so any PR with a compute modification will failed until someone fixes this bug.

If you have the opportunity to fix #352, it will be awesome :). You can push it here at the same time.

Thank you,

**Comment by lvh** [comment]
Thanks for the heads up @lmazuel. I thought the clients were generated from this Swagger. Is that not true, or has this caused a halt in the development of e.g. the Python Azure library?

**Comment by lmazuel** [comment]
@lvh We added a few new verifications of the Swagger validity and found a problem in the compute file. Since this new version of Autorest, the Python extension is no more able to generate the compute client.
Please note that @tbombach is currently working on a generic linting system and that at short term all clients (C#, Java, etc.) might be considered generated from an invalid file. If you have any way to push a fix of this bug, it would be great!
Thank you,

**Comment by lmazuel** [comment]
@lvh https://github.com/Azure/azure-rest-api-specs/pull/359

**Comment by John-Hart** [comment]
@lvh the Travis-ci issue has been fixed with  #359 can you pull from upstream and push your changes again?

**Comment by lvh** [comment]
I'm working on getting the CLA signed; I have re-applied the fixes to compute on top of master.

**Comment by azurecla** [comment]
Hi __@lvh__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by lvh** [comment]
(Oops; misclicked -- I did not intend to do that.)

File: arm-commerce/2015-06-01-preview/swagger/commerce.json [hunk 1]
```diff
@@ -116,7 +116,7 @@
             "in": "query",
             "required": false,
             "type": "string",
-            "description": "The filter to apply on the operation. It ONLY supports the �eq� and �and� logical operators at this time. All the 4 query parameters 'OfferDurableId',  'Currency', 'Locale', 'Region' are required to be a part of the $filter."
```
Comment (lvh) [inline arm-commerce/2015-06-01-preview/swagger/commerce.json]: I have no idea why the diff generator thinks `eq` changed but `and` didn't :)

**Comment by azurecla** [comment]
__@lvh__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

File: arm-commerce/2015-06-01-preview/swagger/commerce.json (hunk 1 above)
Comment (John-Hart) [inline arm-commerce/2015-06-01-preview/swagger/commerce.json]: @lvh It's showing both 'eq' and 'and' changed, is that not what you expected?

Comment (lvh) [inline arm-commerce/2015-06-01-preview/swagger/commerce.json]: Yes, I changed both of the quotes. But the diff generator thinks the _word_ itself changed, which it didn't. I checked in the commit diff; it's just a GitHub display issue.

**Comment by John-Hart** [comment]
Looks Good

**Pull request merged**